miqcli/api.py

```diff
@@ -276,8 +276,16 @@ def __init__(self, conf, verbose=False):
         :type verbose: bool
         """
 
+        # lets first load the correct server configuration settings
+        config = Config(defaults=DEFAULT_CONFIG, verbose=verbose)
```

Comment:
`defaults=` does not match with what it is. This is actual `settings` or something else, which, by default, receiveds DEFAULT_CONFIG. but I would call it something other than `defaults`. Does it make sense?

Reply:
Yes that does make sense. I would prefer `settings` over `defaults`. That parameter name has been set and I was just re-using it. Let me make an update to modify this parameter name to better map to what it actual is accepting.